### PR TITLE
Rename instances of "livestreamer" to "streamlink"

### PR DESCRIPTION
**Pull request by cdrage**
Renames all instances of "livestreamer" to "streamlink".

### CODE REVIEW

**Comment by cdrage** [comment]
Tested with Twitch and all seems to pass ^^

File: src/streamlink/__init__.py [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
+"""Streamlink extracts streams from various services.
 
-The main compontent of Livestreamer is a command-line utility that
+The main compontent of Streamlink is a command-line utility that
 launches the streams in a video player.
 
 An API is also provided that allows direct access to stream data.
 
-Full documentation is available at http://docs.livestreamer.io/.
+Full documentation is available at http://docs.streamlink.io/.
```
Comment (gravyboat) [inline src/streamlink/__init__.py]: We should probably modify this at some point to either use github pages, or read the docs to ensure it stays as openly accessible as possible.

File: src/streamlink/plugin/plugin.py [hunk 2]
```diff
@@ -165,7 +165,7 @@ def func(*args, **kwargs):
             )
 
             if issue:
-                msg += "More info: https://github.com/chrippa/livestreamer/issues/{0}".format(issue)
+                msg += "More info: https://github.com/chrippa/streamlink/issues/{0}".format(issue)
```
Comment (gravyboat) [inline src/streamlink/plugin/plugin.py]: This needs the new repo URL

**Comment by cdrage** [comment]
Yay tests pass! :sparkles:

File: src/streamlink_cli/argparser.py [hunk 3]
```diff
@@ -136,15 +136,15 @@ def keyvalue(value):
...
     usage="%(prog)s [OPTIONS] [URL] [STREAM]",
     description=dedent("""
-    Livestreamer is command-line utility that extracts streams from
+    Streamlink is command-line utility that extracts streams from
     various services and pipes them into a video player of choice.
     """),
     epilog=dedent("""
     For more in-depth documention see:
-      http://docs.livestreamer.io/
+      http://docs.streamlink.io/
```
Comment (gravyboat) [inline src/streamlink_cli/argparser.py]: Another doc URL instance.

File: src/streamlink_cli/argparser.py [hunk 4]
```diff
@@ -136,15 +136,15 @@ def keyvalue(value):
...
 
     Please report broken plugins or bugs to the issue tracker on Github:
-      https://github.com/chrippa/livestreamer/issues
+      https://github.com/chrippa/streamlink/issues
```
Comment (gravyboat) [inline src/streamlink_cli/argparser.py]: Another repo URL instance.

File: src/streamlink/__init__.py (hunk 1 above)
Comment (sbstp) [inline src/streamlink/__init__.py]: @gravyboat I opened an issue regarding the web page, #14.

**Comment by cdrage** [comment]
@gravyboat thanks man for the review! didn't think it would have chrippa still in there.

I'll go ahead and update this PR.

**Comment by cdrage** [comment]
@gravyboat updated!

**Comment by gravyboat** [comment]
Looks good, merging!

**Pull request merged**